src/runtime_src/core/tools/xbmgmt2/xbmgmt-csh-completion

```diff
@@ -90,6 +93,12 @@ else
     case "--image":
       set programOptions=""
       breaksw
+    case "--input":
```

Comment:
Future thought
It would be great if xbmgmt / xbutil as part of the CMake build would produce an output that is "completion" friendly.